node/testnet/src/chain_spec.rs

```diff
@@ -162,14 +164,12 @@ fn testnet_genesis(
 			// Assign network admin rights.
 			key: root_key,
 		}),
-		pallet_generic_asset: Some(GenericAssetConfig{
-			assets: vec![0],
-			initial_balance: UNIT*UNIT,
+		polkadex_custom_assets: Some(CustomAssetConfig{
+			assets: vec![H256::zero()],
```

Comment:
@ksr30 Instead of Zero we should use 
```suggestion
			assets: vec![H256::random()],
```

Reply:
@ksr30 If I remember it correctly, its function name is random but look into the available functions of H256 for random.
We need to use a random asset ID rather than zero